src/Wizard/Wizard.jsx

```diff
@@ -18,22 +18,31 @@ export class Wizard extends React.Component {
     this.state = _.assign(INITIAL_STATE, {
       data: props.initialWizardData || {},
     });
+
     this.reset = this.reset.bind(this);
     this.prevStepHandler = this.prevStepHandler.bind(this);
     this.nextStepHandler = this.nextStepHandler.bind(this);
     this.calculatePercentComplete = this.calculatePercentComplete.bind(this);
+
+    if (this.props.initialStep) {
+      this.state = Object.assign({}, this.state, this._getStateAtStep(this.props.initialStep));
+    }
   }
 
   reset() {
     this.setState(_.assign({}, INITIAL_STATE, {data: {}}));
   }
 
-  jumpToStep(idx) {
-    this.setState({
+  _getStateAtStep(idx) {
+    return {
       currentStep: idx,
-      stepsVisited: _.union(this.state.stepsVisited, [idx]),
+      stepsVisited: _.range(idx + 1), // include the step itself
```

Comment:
I know this goes counter to the discussion we just had in person, but we probably don't want to break the pre-existing behavior of `jumpToStep`, which is used when `seekable` is true. So maybe we just need to keep `jumpToStep` and the constructor for `initialStep` totally separate? Up to you so long as there isn't a breaking change.